### DISCUSSION TITLE
pyz archive

### DISCUSSION THREAD

**Discussion by AntonioMembrino** [extremecoders-re/pyinstxtractor#42] [General]
hi,
I have a pyz archive (I don't know the python version) I'd like to extract the files from it. Is it possible?

**Comment by extremecoders-re** [comment]
pyinstxtractor should work with pyz archive files. Just run the script as you would do with an executable.

Or you can try the archive_viewer utility which is included with PyInstaller itself.
https://pyinstaller.org/en/stable/advanced-topics.html#using-pyi-archive-viewer
